pkg/controller/v1alpha2/applicationconfiguration/applicationconfiguration.go

```diff
@@ -72,16 +72,16 @@ const (
 )
 
 // Setup adds a controller that reconciles ApplicationConfigurations.
-func Setup(mgr ctrl.Manager, l logging.Logger) error {
+func Setup(mgr ctrl.Manager, args controller.Args, l logging.Logger) error {
 	name := "oam/" + strings.ToLower(v1alpha2.ApplicationConfigurationGroupKind)
 
 	return ctrl.NewControllerManagedBy(mgr).
 		Named(name).
 		For(&v1alpha2.ApplicationConfiguration{}).
 		Watches(&source.Kind{Type: &v1alpha2.Component{}}, &ComponentHandler{
-			Client:     mgr.GetClient(),
-			Logger:     l,
-			AppsClient: clientappv1.NewForConfigOrDie(mgr.GetConfig()),
```

Comment:
It seems AppsClient was removed? This is issue also doing this work. https://github.com/crossplane/oam-kubernetes-runtime/issues/152